content/paths/sign_requests__get_sign_requests.yml

```diff
@@ -0,0 +1,34 @@
+---
+operationId: get_sign_requests
+
+summary: List Sign Requests
+
+tags:
+  - Sign Requests
+
+x-box-tag: sign_requests
+
+description: |-
+  Gets Sign Requests created by a user.
+
+parameters:
+  - $ref: '../attributes/offset.yml'
```

Comment:
Unfortunately since we're only wrapping the Sign Request endpoints that exist we have to work with what they currently support and from talking to their team it seems like they can only support marker based pagination currently

Reply:
In conversation with the sign team now to see if they can support marker based pagination so this is TBD on that

Reply:
As an aside here, since markers are opaque, it's likely possible to implement a marker-based API with a hidden offset in the marker, and follow up with better markers and no breakage.

Reply:
The sign team added implementation for marker based pagination, so updated this